src/components/button/AposButton.vue

```diff
@@ -91,6 +114,11 @@ export default {
     }
   }
 
+  .apos-button--hollow,
```

Comment:
"hollow" doesn't make total sense to me, but not a biggie.

Reply:
"clear"?

Reply:
hmm i thought hollow and ghost we're pretty common .. looks like Zurb calls them hollow, bootstrap calls them outline. I think i like outline of these

Reply:
Ah, I never really spent any time with Foundation.